plugin.xml

```diff
@@ -27,6 +27,8 @@
         compile 'org.jboss.aerogear:aerogear-android-core:2.0.0-alpha.1@aar'
         compile 'org.jboss.aerogear:aerogear-android-pipe:2.0.0-alpha.1@aar'
         compile 'org.jboss.aerogear:aerogear-android-store:2.0.0-alpha.1@aar'
+        compile 'com.google.code.gson:gson:1.7.2'
+        compile 'com.google.guava:guava:18.0'
```

Comment:
ah, ok. Instead of temporary adding this, perhaps we can wait a bit until 2.0.0 is available via maven central?